src/components/atoms/Plugin/hooks.ts

```diff
@@ -114,6 +116,22 @@ export default function useHook({
     [messageEvents, messageOnceEvents, onError, rawOnMessage],
   );
 
+  const eventLoopCb = useCallback(() => {
+    if (!arena.current) return;
+    try {
+      arena.current.executePendingJobs();
+      if (arena.current.context.runtime.hasPendingJob()) {
+        eventLoop.current = window.setTimeout(eventLoopCb, 0);
+      }
+    } catch (err) {
+      onError(err);
+    }
+  }, [onError]);
+
+  const startEventLoop = useCallback(() => {
```

Comment:
I pass this to `exposed()` so that can trigger a event loop check on promise `finally`.